.github/workflows/on_pr_closed.yaml

```diff
@@ -2,19 +2,20 @@ name: Uninstall PR instance
 on:
   pull_request:
     branches:
-      - main
+      - ghcr-clean
```

Comment:
This is incorrect, PRs should be targeting `main` (as it was set to do before).

Reply:
Hi @esune i am testing stuff so running on my branch